internal/csi-addons/rbd/replication.go

```diff
@@ -746,8 +746,8 @@ func (rs *ReplicationServer) ResyncVolume(ctx context.Context,
 }
 
 // timestampToString converts the time.Time object to string.
-func timestampToString(st *timestamppb.Timestamp) string {
-	return fmt.Sprintf("seconds:%d nanos:%d", st.GetSeconds(), st.GetNanos())
+func timestampToString(st *time.Time) string {
+	return fmt.Sprintf("seconds:%d nanos:%d", st.Unix(), st.Nanosecond())
```

Comment:
can we check on this one, Does Unix is equivalent to GetSeconds()?

Reply:
Yes
- https://pkg.go.dev/google.golang.org/protobuf/types/known/timestamppb#Timestamp
- https://pkg.go.dev/time#Time.Unix